packages/launch/src/focused-launch/success/index.tsx

```diff
@@ -39,6 +51,19 @@ const Success: React.FunctionComponent = () => {
 	const [ displayedSiteUrl, setDisplayedSiteUrl ] = React.useState( '' );
 	const [ hasCopied, setHasCopied ] = React.useState( false );
 
+	const isEcommercePlan = useSelect(
+		( select ) => {
+			const plansStore = select( PLANS_STORE );
+			const plan = plansStore.getPlanByProductId( planProductId, locale );
+			return plansStore.isPlanEcommerce( plan?.periodAgnosticSlug );
+		},
+		[ planProductId, locale ]
+	);
+
+	// if the user has an ecommerce plan or they're using focused launch from wp-admin
+	// they will be automatically redirected to /checkout, in which case the CTAs are not needed
+	const willUserBeRedirectedAutomatically = ! isInIframe || isEcommercePlan;
```

Comment:
Since this is the same logic used in https://github.com/Automattic/wp-calypso/blob/b5d9d3d4fb238ade25b5bb61dfe36b20597a2012/packages/launch/src/hooks/use-cart.ts#L78, maybe we could extract it to a reusable hook?

Leaving it as is may lead to inconsistencies in the future, in case we update one file and forget to update the other.

Reply:
Very good point.